src/includes/class-wp-cli.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Network Management Command
+ *
+ */
+class WPMultiNetwork_Command extends WP_CLI_Command {
+
+	/**
+	 * @var array $obj_fields Default fields to display for each object.
+	 */
+	protected $obj_fields = array(
+		"id",
+		"domain",
+		"path"
+	);
+
+	/**
+	 * Add a Network
+	 *
+	 * <domain>
+	 * : Domain for network
+	 *
+	 * <path>
+	 * : Path for network
+	 *
+	 * [--site_name=<site_name>]
+	 * : Name of new network
+	 *
+	 * [--clone_network=<clone_network>]
+	 * : ID of network to clone
+	 *
+	 * [--options_to_clone=<options_to_clone>]
+	 * : Options to clone to new network
+	 *
+	 */
+	public function create( $args, $assoc_args ) {
+
+		list ( $domain, $path ) = $args;
```

Comment:
Yeah, the extra space after the `list` is just irking me. Not a huge issue though. :)